requirements/analyze.txt

```diff
@@ -38,7 +38,6 @@ iso8601==0.1.12           # via -r requirements/requirements.txt, colander, defo
 isort==5.5.1              # via pylint
 itsdangerous==1.1.0       # via -r requirements/requirements.txt
 jinja2==2.11.2            # via -r requirements/requirements.txt, pyramid-jinja2
-jsonpointer==2.0          # via -r requirements/requirements.txt
```

Comment:
I updated these files by hand since I couldn't see a `make` command that ran `pip-compile`. That should be OK since I'm just removing an entry. What is the correct way to update all of these files now though?

Reply:
I think you are expected to run pip-compile inside the target environment now: https://stackoverflow.com/c/hypothesis/questions/96